### PR TITLE
Add remove features

### PR DESCRIPTION
**Pull request by MenendezON**
This pull request concern the update of our former Webpack project. In this version, you'll find :
- Adding function
- Removing function
- Editing function
- Storage data

### CODE REVIEW

File: src/function.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/* eslint-disable-next-line import/no-cycle */
```
Comment (V-Blaze) [inline src/function.js]: - - You have done great so far. adding, removing, and editing tasks works great. :+1: :100:  But there are some changes you need to make before moving to the next milestone. :smile: 
- [ ]  As required by Microverse, you are not meant to use any linters disable rule. The linters are setup to make sure you follow the best industry standard practices. Therefore I'll suggest you remove the disable rule and fix the issue. :smile: :wave:

File: src/index.js [hunk 2]
```diff
@@ -1,28 +1,21 @@
...
-];
+let tasks = [];
+
+export const main = document.querySelector('.container');
+const dataLoading = () => {
+  tasks = JSON.parse(localStorage.getItem('datas')) ?? [];
+  main.innerHTML = '';
+  /* eslint-disable */
+  document.body.appendChild(component());
+  /* eslint-enable */
```
Comment (V-Blaze) [inline src/index.js]: - [ ] Just as I pointed out above, I'll suggest you remove these linters disable rules and try to re-write the logic of the code to fix the issue. :wave: :+1:

File: src/remove.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const removeTask = (i) => {
+  let tasks = JSON.parse(localStorage.getItem('datas'));
+  tasks = tasks.filter((task) => task.description !== tasks[i].description);
+  /* eslint-disable no-plusplus */
```
Comment (V-Blaze) [inline src/remove.js]: - [ ] I'll suggest you do the same thing here and remove the linters disable. :waffle: :+1:  You can replace the `i++` with `i += 1` to resolve this issue here. :+1:

File: dist/index.html [hunk 4]
```diff
@@ -2,10 +2,11 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <!-- <meta name="viewport" content="width=device-width, initial-scale=1.0"> -->
```
Comment (V-Blaze) [inline dist/index.html:5]: - [ ] It's against the best practices to leave unused codes as comments on your codebase. I'll suggest you remove this. :+1: :+1:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @MenendezON , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Add, remove and edit feature works great✔
-  LInteers setup correctly👋 
-  Added a new file for the new functionality✔

 ## Required Changes ♻️
- [ ] Currently, the MIT LICENSE link doesn't lead to the correct page of this repository, it leads to a 404 error page. I'll suggest you fix this as broken links in your readme makes it look a bit unprofessional. :+1: 
- - Screenshot:
Screenshot from 2023-07-10 14-31-38


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 5]
```diff
@@ -1,65 +1,87 @@
...
-    completed: false,
-    index: 3,
-  },
-];
+let tasks = [];
+export const main = document.querySelector('.container');
+const dataLoading = () => {
+  tasks = JSON.parse(localStorage.getItem('datas')) ?? [];
+  main.innerHTML = '';
+  /* eslint-disable */
```
Comment (nedjwab) [inline src/index.js:12]: [ ] As the previous reviewer mentions, you have to remove these linters disable rules, and try to re-write the logic of the code to fix the issue. 👋 👍

**Review by nedjwab — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @MenendezON 

Nice job, you are almost there.



## (Highlights) Good Points: :
- Good implementation of some of the project requirements  ✔
- GitHub flow was correctly used ✔.

## (Changes Required) Aspects to improve: ♻️
You did great implementing some of the project requirements, but you still have some issues: 
check the comments below 
 👇👇👇

 
## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**